src/App.test.tsx

```diff
@@ -3,13 +3,26 @@ import { render, screen } from '@testing-library/react';
 import App from './App';
 import { ThemeProvider } from 'styled-components'
 import theme from './theme'
+import { Provider } from 'react-redux'
+import configureMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
+import { selectCurrentUser, selectAuthLoading } from './store/slices/authentication/selectors'
 
-test('renders the welcome text', () => {
-  render(
-    <ThemeProvider theme={theme}>
-      <App />
-    </ThemeProvider>
-  );
-  const welcomeElement = screen.getByText('Happy coding!');
-  expect(welcomeElement).toBeInTheDocument();
-});
+const mockStore = configureMockStore([thunk])
+const store = mockStore({})
+
+jest.mock('./store/slices/authentication/selectors', () => ({
+  selectCurrentUser: jest.fn(),
```

Comment:
I think we should either mock out redux with the useSelectors and then let us not provide a mockstore, or we should provide a mockstore with real values, and then not mock out the useSelector.